features/todo_list.feature

```diff
@@ -50,6 +54,9 @@ Feature:
     And the file ".todo.reek" should contain:
       """
       ---
+      InstanceVariableAssumption:
+        exclude:
+        - Smelly
```

Comment:
Not blocking for this PR but we really need to fix our [test set up](https://github.com/troessner/reek/issues/978) before going on with the next smell detector. This is barely maintainable right now.